google/cloud/aiplatform/prediction/model_server.py

```diff
@@ -189,7 +189,7 @@ def set_number_of_workers_from_env() -> None:
 if __name__ == "__main__":
     cpr_user_dir_name = os.getenv("CPR_USER_DIR_NAME")
     if cpr_user_dir_name:
-        sys.path.insert(0, os.path.join(os.getcwd(), cpr_user_dir_name))
+        sys.path.append(os.path.join(os.getcwd(), cpr_user_dir_name))
```

Comment:
Can we insert in the second position after /usr/app? I think we should search through user provided src code first before searching through other installed dependencies.

Reply:
Done.